src/blocks/blocks/flip/types.d.ts

```diff
@@ -1,24 +1,25 @@
-import { BlockProps, ImagePosition, InspectorProps, MediaImageProps } from '../../helpers/blocks';
+import { BlockProps, BorderType, ImagePosition, InspectorProps, MediaImageProps, PaddingType, Responsive } from '../../helpers/blocks';
 
 type Attributes = {
 	id: string
 	isInverted: boolean
 	title: string
 	description: string
 	animType: string
-	width: number
-	height: number
-	padding: number
+	width: number | Responsive<string>
+	height: number | Responsive<string>
+	padding: number | Responsive<PaddingType>
```

Comment:
@Soare-Robert-Daniel It's just better to use different attributes for Tablet and Mobile at this point in case we ever need to do a deprecation or WordPress introduces new ways to handle this.